src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -964,4 +895,103 @@ public void setNumOfRubricSubQuestions(int numOfRubricSubQuestions) {
         return rubricSubQuestions;
     }
 
+    /**
+     * Class to calculate the statistics of responses for a rubric question
+     */
+    private static class RubricStatistics {
```

Comment:
I did not consider taking out the questionDetails object at first so I thought static nested was a better choice as it shows that the RubricStatistics class did not depend on other variables of the FeedbackRubricQuestionDetails object calling it. 
I just realised that it is possible to construct a inner class object from questionDetails object instead but the syntax is not very nice and potentially confusing.
```
RubricStatistics statistics = fqd.new RubricStatistics(responsesForStatistics);
```